pkg/ssh/ssh.go

```diff
@@ -371,17 +371,17 @@ func (l *SSHTunnelList) removeAndReAdd(e sshTunnelEntry) {
 	}
 	l.tunnelsLock.Unlock()
 	if err := e.Tunnel.Close(); err != nil {
-		klog.Infof("Failed to close removed tunnel: %v", err)
+		klog.InfoS("Failed to close removed tunnel: ", err)
 	}
 	go l.createAndAddTunnel(e.Address)
 }
 
 func (l *SSHTunnelList) Dial(ctx context.Context, net, addr string) (net.Conn, error) {
 	start := time.Now()
 	id := mathrand.Int63() // So you can match begins/ends in the log.
-	klog.Infof("[%x: %v] Dialing...", id, addr)
+	klog.InfoS("[", id, ":", addr, "]", "Dialing...")
```

Comment:
This is not a proper way to use InfoS interface.

Reply:
thanks